app/Http/Controllers/ChartsController.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\CRequest;
+
+class ChartsController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+       
+        return view('admin.charts.index');
+    }
+
+    public function week($week) 
+    {
+        $requests = CRequest::where('is_approve', true)->get();
+        $requestsOfWeek = [];
+        foreach($requests as $request) {
```

Comment:
```suggestion
        foreach ($requests as $request) {
```
sửa hết mấy chỗ tương tự